### ISSUE TITLE
Use cgcloud to terminate instances in adam_überscript

### ISSUE THREAD

**Issue by fnothaft** [BD2KGenomics/toil-scripts#182]
Right now, we're using Boto, but this isn't blocking. We should use cgcloud instead after #181 merges.

**Comment by hannes-ucsc** [comment]
https://github.com/BD2KGenomics/cgcloud/issues/128 and PR https://github.com/BD2KGenomics/cgcloud/issues/129

Let's include this in #181 because how #181 currently works around the asynchrony of instance termination is expensive and ineffective.

**Comment by hannes-ucsc** [comment]
BD2KGenomics/cgcloud#129 has been merged.

**Comment by hannes-ucsc** [comment]
I'd be ok to leave this out of #181 and take care of this in a separate PR. Your call, @fnothaft.
